controllers/playersIndexController.js

```diff
@@ -0,0 +1,29 @@
+const db = require('../db');
+
+// GET http://localhost:7000/admin/players
+exports.getIndex = (req, res, next) => {
+    console.log("GET: Hello getPlayersPage")
+    // connect to the database and save the new incoming player
+    db.getDb()
+        .db('softball')
+        // the collction will be created dynamically if it does not exist yet
+        .collection('players')
+        .find(
```

Comment:
Actually, `find()` without parameters will do the same thing. https://docs.mongodb.com/manual/reference/method/db.collection.find/

But it was a good idea to explicitly put this options because it gives the idea of the data type for a new engineer (that the data have "name" and "email" just by looking at the code) 
great 👍

Reply:
Now I get 500 error...;(
although it's connected with db and I can see console.log `GET: Hello getPlayersPage`


![Screenshot 2021-09-07 at 17 23 28](https://user-images.githubusercontent.com/60405643/132370709-064d269f-ac10-4c97-81f1-3a8e4cf55e31.png)

Reply:
ah, ok. The callback function was `function(err, result)`. We changed it to `function(result)`. Now `result` became `err`. Let's keep `err` as it was before and just not use it.